examples/nextjs/flatbread.config.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  markdownTransformer,
+  filesystem,
+  type FlatbreadConfig,
+} from 'flatbread';
+
+const transformerConfig = {
+  markdown: {
+    gfm: true,
+    externalLinks: true,
+  },
+};
+const config = {
+  source: filesystem(),
+  transformer: markdownTransformer(transformerConfig),
+
+  content: [
+    {
+      path: '../../packages/flatbread/content/posts',
```

Comment:
We could add a folder in the root containing a set of generated data, if it'll work with AVA